src/pages/dataspace/[dataspace-id]/dataset/[dataset-id].tsx

```diff
@@ -0,0 +1,72 @@
+import type { UseState } from '@okp4/ui'
+import type { NextPage } from 'next'
+import { useEffect, useState } from 'react'
+import {
+  DatasetPreview,
+  DatasetInformation,
+  PageTitle,
+  PreviousPageButton
+} from '../../../../components'
+import './datasetId.scss'
+import router from 'next/router'
+import type { ParsedUrlQuery } from 'querystring'
+
+export type DatasetGovernance = {
+  readonly name: string
+  readonly based: string
+}
+
+type Access = 'PRIVATE' | 'PUBLIC'
+
+export type Dataset = {
+  readonly id: string
+  readonly mainPicture: string
+  readonly name: string
+  readonly type: string
+  readonly access: Access
+  readonly categories: Array<string>
+  readonly description: string
+  readonly provider: string
+  readonly governance: DatasetGovernance
+  readonly size: number
+  readonly format: string
+  readonly quality: number
+  readonly completeness: number
+  readonly updatedOn: string
+}
+
+
+const fetchDataset = async (url: string): Promise<Dataset> => {
+  const response = await fetch(url)
+
+  if (response.status !== 200) {
+    throw new Error(response.statusText)
+  }
+
+  const dataset: Promise<Dataset> = response.json()
+  return dataset
+}
+
+const DatasetId: NextPage = () => {
+  const [dataset, setDataset]: UseState<Dataset | null> = useState<Dataset | null>(null)
+  const query: ParsedUrlQuery = router.query
+
+  useEffect(() => {
+    if (query['dataset-id']) {
+      fetchDataset('/api/fake/dataset/ef347285-e52a-430d-9679-dcb76b962ce7')
```

Comment:
Why you don't use the `dataset-id`?
```suggestion
      fetchDataset(`/api/fake/dataset/${query.dataset-id}`)
```

Reply:
I don't use `dataset-id` because I don't have all the fake data. I thought we were waiting the full fake data to implement the fetch of the requested data.
I can replace the query with the query params but this page will work only with one url (beacause we only have one fake data atm).

Reply:
OK
But IMHO it's better to use a query param, even if it's fake ;)

Reply:
Ok, i'll patch it and update the PR description with the valid url !